packages/pixel-demo/test/unitTests/pixels/types/test-pixel-list.js

```diff
@@ -216,7 +216,7 @@ test('pixelList without', t => {
   ]);
   t.throws(
     () => extentOps.without(harden([]), harden([startPixel])),
-    /part is not in whole/,
+    /\(an object\) must be in \(an object\)/,
```

Comment:
This change seems to make our tests less intuitive and helpful. Can we add some context back in since the error message doesn't have the objects?

Reply:
added more useful text to the error message.